src/pages/dataTableUtilities/getIndicatorTableData.js

```diff
@@ -172,7 +59,12 @@ const mapIndicatorTableRow = (row, period) => {
  * @param {Period} period
  * @return {Array.<object>}
  */
-const mapIndicatorTableRows = (rows, period) => rows.map(row => mapIndicatorTableRow(row, period));
+const mapIndicatorTableRows = (rows, period) => {
```

Comment:
having the `get` simply call this `map` is pretty redundant !

Reply:
Yeah can't remember why this way done this way. I think it was another testing stub. ac0b5f3465f6e3ff6af6b5064ac36f2c7185006f.